# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1732918482 -3600
# Parent  6cb29c5ad9071343e305cacc165a3caac0f82b99
Branding: Use about logo also in add-on debug page.

diff --git a/mail/themes/shared/jar.inc.mn b/mail/themes/shared/jar.inc.mn
--- a/mail/themes/shared/jar.inc.mn
+++ b/mail/themes/shared/jar.inc.mn
@@ -15,9 +15,9 @@
 # Devtools overrides
 % override chrome://devtools/skin/images/aboutdebugging-firefox-nightly.svg chrome://branding/content/about-logo.svg
 % override chrome://devtools/skin/images/aboutdebugging-firefox-release.svg chrome://branding/content/about-logo.svg
 % override chrome://devtools/skin/images/aboutdebugging-firefox-beta.svg chrome://branding/content/about-logo.svg
-% override chrome://devtools/skin/images/aboutdebugging-firefox-logo.svg chrome://messenger/skin/icons/aboutdebugging-logo.svg
+% override chrome://devtools/skin/images/aboutdebugging-firefox-logo.svg chrome://branding/content/about-logo.svg
 
 # Proton icon overrides
 % override chrome://global/skin/icons/add.svg chrome://messenger/skin/overrides/add.svg
 % override chrome://global/skin/icons/blocked.svg chrome://messenger/skin/overrides/blocked.svg
